src/components/icons/RefreshIcon.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Refresh SVG Icon
+ *
+ * @param {object} props  Data passed to the component
+ * @param {number} props.size  Width and height of the icon
+ * @param {string} props.className  Additional className for the component
+ * @param {object} props.style  Inline style object for the component
+ * @returns {object} JSX of the icon
+ *
+ */
+export default function RefreshIcon({size, className, style}) {
+  const {fill} = style || {};
+
+  return (
+    <svg width={size || 24} height={size || 24} xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" className={`wxc-icon ${className}`} style={{fill}}>
```

Comment:
Is there a refresh icon in momentum that we could use? Does this refresh icon match what we have on the Webex App?

Reply:
We are not using momentum icons in the components repo.